api/dataservices/fdoprofile/fdoprofile.go

```diff
@@ -67,12 +67,10 @@ func (service *Service) FDOProfile(ID portainer.FDOProfileID) (*portainer.FDOPro
 
 // Create assign an ID to a new FDO Profile and saves it.
 func (service *Service) Create(FDOProfile *portainer.FDOProfile) error {
-	return service.connection.CreateObject(
+	return service.connection.CreateObjectWithId(
```

Comment:
I noticed that the profiles were being created with id sequence 2,4,6,8.... this fixes it